server/app/views/admin/programs/ProgramNewOneView.java

```diff
@@ -37,7 +38,9 @@ public ProgramNewOneView(
 
   /** Renders the create form. */
   public Content render(Request request) {
-    return render(request, new ProgramForm(), Optional.empty(), Optional.empty());
+    ProgramForm programForm = new ProgramForm();
+    programForm.setNotificationPreferences(ProgramNotificationPreference.getDefaultsForForm());
```

Comment:
Good question, I added this comment:
```
    // We set this default here, instead of in the ProgramForm constructor, because otherwise
    // the setting gets re-enabled when the form is reloaded after an error. This is because
    // an unset checkbox value is not included in the POST at all, so there's nothing to
    // override the value the constructor initializes.
```